### PR TITLE
[feature/MARVP-42] event feedback animation

### PR DESCRIPTION
**Pull request by krishk95**

### CODE REVIEW

File: actions/mobile/appEvents.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+define({
+    AS_AppEvents_h334c68ef261418898ed0881d70a84cb: function AS_AppEvents_h334c68ef261418898ed0881d70a84cb(eventobject) {
+        var self = this;
+        var appEventData = accelerateEventData.eventdata;
+        var updatedData = {
+            "data": "Sample Test"
```
Comment (mishradileep) [inline actions/mobile/appEvents.js]: Can we put some meaningful text here. Get Siddhanta's help if required.

File: controllers/mobile/frmFeedbackController/frmFeedbackController.js [hunk 2]
```diff
@@ -79,9 +80,174 @@ define({
...
+
+    /** 
+    	* @function animateFeedbackFields
+        * @description This function is used to animate all the feedback fields
+        				from bottom to top one after another
+        * @private
+    */
+    animateFeedbackFields: function() {
+        var self = this;
+        this.animateFeedbackPrompt("100dp", false);
```
Comment (mishradileep) [inline controllers/mobile/frmFeedbackController/frmFeedbackController.js]: A function doing nothing but calling another function. Can we remove this

File: controllers/mobile/frmFeedbackController/frmFeedbackController.js [hunk 3]
```diff
@@ -79,9 +80,174 @@ define({
...
+
+    /** 
+    	* @function closeFeedbackForm
+        * @description This function is used to animate all the feedback fields
+        				from top to bottom one after another at the time of form close
+        * @private
+    */
+    closeFeedbackForm: function() {
+        var self = this;
+        this.animateSubmitButton("100%", true);
```
Comment (mishradileep) [inline controllers/mobile/frmFeedbackController/frmFeedbackController.js]: Same as above

**Review by mishradileep — CHANGES_REQUESTED** [review]
Need some serious attention on code formatting/indentation. Also, check comments inline

**Review by vigneshwarankarunanithi — APPROVED** [review]

**Review by vijaysamanthapudi — APPROVED** [review]

**Pull request merged**